qiskit_algorithms/gradients/base/base_estimator_gradient.py

```diff
@@ -107,10 +101,12 @@ def run(
                 ``circuits``. Defaults to None, which means that the gradients of all parameters in
                 each circuit are calculated. None in the sequence means that the gradients of all
                 parameters in the corresponding circuit are calculated.
-            options: Primitive backend runtime options used for circuit execution.
-                The order of priority is: options in ``run`` method > gradient's
-                default options > primitive's default setting.
-                Higher priority setting overrides lower priority setting
+            precision: Precision to be used by the underlying Estimator. If a single float is
+                provided, this number will be used for all circuits. If a sequence of floats is
+                provided, they will be used on a per-circuit basis. If none is provided, the
```

Comment:
> If none is provided, the
> gradient's default precision will be used for all circuits. If this number is
> also set to None, the underlying primitive's default precision will be used
> for all circuits.

I am trying to parse these - why are there two sentences and `none` in the first case I guess that means `None` or is that somehow an empty sequence? The "also" in the second sentence I guess I parse "as well as" which makes things hard for me to read/comprehend. Either which way the first sentence, if it has meaning, and None as per the second just mean the `the underlying primitive's default precision will be used for all circuits.` either way.

Reply:
I meant `none` in the sens of `None` indeed, it's missing the capitalization. Just to be sure I understand, does the following work for you?
`precision: Precision to be used by the underlying Estimator. If a single float is
                provided, this number will be used for all circuits. If a sequence of floats is
                provided, they will be used on a per-circuit basis. If not set, the
                gradient's default precision will be used for all circuits. If this number is
                also not set the underlying primitive's default precision will be used
                for all circuits.`

Reply:
In the last part `If this number is also not set...` I guess the `this` refers to the `gradients default precision` number in the prior sentence. So if I do it like below is that the intent (where I guess if I pass a specific precision to the estimator that is used by default - hence I put brackets around the default since it will only use its own internal default, I think, in the case nothing is provided).

`If not set, the gradient's default precision will be used for all circuits., and if that is None (not set) then the underlying primitive's (default) precision will be used for all circuits.`